src/components/dropdown/revo-dropdown.style.scss

```
@@ -30,15 +30,19 @@ revo-dropdown {
     flex-wrap: wrap;
     position: relative;
     border-radius: 4px;
+    cursor: pointer;
 
-    cursor: text;
     display: inline-flex;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)